go.mod

```diff
@@ -131,6 +131,7 @@ require (
 	github.com/hashicorp/go-cleanhttp v0.5.1 // indirect
 	github.com/hashicorp/go-retryablehttp v0.7.1 // indirect
 	github.com/jmespath/go-jmespath v0.4.0 // indirect
+	github.com/kr/pretty v0.3.0 // indirect
```

Comment:
@AkihiroSuda I'm curious now where this one came from, because I just updated my PR for update containerd (which also includes the runc 1.1.5 update), and I didn't see it appear there;

- https://github.com/moby/buildkit/pull/3736

Reply:
```console
$ go mod why github.com/kr/pretty
# github.com/kr/pretty
github.com/moby/buildkit/client/ociindex
github.com/gofrs/flock
github.com/gofrs/flock.test
gopkg.in/check.v1
github.com/kr/pretty
```

Reply:
Looks like this is not a real dependency for BuildKit, but this is picked by dependabot, and somehow kept by `go mod tidy`
🤔

Reply:
What should we do with this?

Reply:
Why us it in this PR if it is not in https://github.com/moby/buildkit/pull/3736/files that does the same change?

Reply:
Yeah, it's still weird; have we tried removing it, and rerunning vendor ?

Reply:
- https://github.com/moby/buildkit/pull/3780
(Cherry-picked into this PR as well)